dev/RELEASE_PROCESS.rst

```diff
@@ -18,17 +18,36 @@ This document describes the process for creating new releases.
    by the script above, but they aren't yet.
 
 4. **Increment the Python version:** Create a PR that increments the Python
-   package version. See `this example`_.
+   package version. See `this example`_. You can use the following to get started,
```

Comment:
Can you update the *Testing* bullet point in point 2 to include the application stress tests?